### DISCUSSION TITLE
Path Completions sometimes breaks

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [mfussenegger/nvim-dap#812] [Q&A]
Is this a known issue ?
When i run dap (with c++ debuger) path completion works fine for the first time,
 if i stop it and start again the path completion fails (inserting `^I`) ?

**Comment by appicon007** [comment]
Came across a similar problem for rust... did you find out any solution? I guess it has something todo with wildchar, wildmenu, wildmode, etc.
